### ISSUE TITLE
Question: Passing annotations to Neo4j standalone chart

### ISSUE THREAD

**Issue by haydenmcp** [neo4j/helm-charts#60]
After downloading the neo4j standalone chart for the first time I attempted to put it in my CircleCI build. I'm using...
- CircleCI
- The [CircleCI Helm Orb](https://circleci.com/developer/orbs/orb/circleci/helm)
- Neo4j Standalone helm chart v4.4.6

I pass the following into the set command [as recommended here](https://github.com/helm/helm/issues/2547):
```
analysis.neo4j.services.neo4j.annotations.service.beta.kubernetes.io/do-loadbalancer-certificate-id=<cert id>
```

as it appears that the services.neo4j.annotations field value is an object. However, I get the following error:

```
Error: UPGRADE FAILED: YAML parse error on thinkdeep/charts/analysis/charts/neo4j/templates/neo4j-loadbalancer.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal object into Go struct field .metadata.annotations of type string
```

Is there something obvious that I'm doing wrong here? Can you provide an example helm set operation usable with the services.neo4j.annotations field? What is it that's expected by the services.neo4j.annotations field? Thanks for your time and help in this matter! :-)

**Comment by harshitsinghvi22** [comment]
Pod annotations feature will be release with 4.4.9 version.

Probably end of next week

**Comment by haydenmcp** [comment]
@harshitsinghvi22 Okay, that's cool. It turns out that the issue I saw was due to the use of helm's set command. Use of special characters caused issues. A workaround that someone mentioned is hard-coding the annotation values in the values.yaml file which worked. It doesn't allow for new helm deployments because you can't set a new cert ID meaning running instances of the deployment in two different environments (i.e, development and production) isn't possible. But as a temporary work-around it's okay. Thanks for the response! Once a new release comes out I'll test it again and see if that makes a difference.

**Comment by harshitsinghvi22** [comment]
@haydenmcp 
Pod annotations and statefulset annotations support is released with 4.4.9
https://github.com/neo4j/helm-charts/releases/tag/4.4.9
